modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/Relationships.tsx

```diff
@@ -33,6 +36,7 @@ interface RelationshipsProps extends IFDSTableProps {
 	baseResourceURL: string;
 	isApproved: boolean;
 	objectDefinitionExternalReferenceCode: string;
+	objectDefinitionId: string | number;
```

Comment:
What type does the backend return?

Reply:
Hey, thanks, this was a temporary measure to have it typed during development, it should be type as `string` only.

Reply:
Will change this in a update.